src/site/content/en/patterns/index.md

```diff
@@ -1,6 +1,5 @@
 ---
 layout: patterns
 title: Patterns
-description: A collection of code snippets to help you optmize your web projects.
----
+description: A collection of code snippets to help you optimize for Web Vitals.
```

Comment:
Patterns are intended not only for Web Vitals, but for other areas as well. Maybe move this description to the /patterns/web-vitals-patterns/index.md?